### PR TITLE
Pm 58 stretch goal daily news feed on website landing page

### PR DESCRIPTION
**Pull request by dotte-osu**
image

### CODE REVIEW

File: src/routes/HomePage/HomePage.js [hunk 1]
```diff
@@ -5,14 +5,17 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
+
+const { HOSTNAME } = require('../../config/hostname.config');
 const log = logUtils.getLogger();
 
+
 export default function HomePage() {
 
   const [pets, setPets] = useState(null);
-  const navigate = useNavigate();
+  const [petNews, setpetNews] = useState(null);
```
Comment (asching7108) [inline src/routes/HomePage/HomePage.js:18]: This is nitpicking but can we change setpetNews to setPetNews?

**Review by asching7108 — COMMENTED** [review]
The news feed section looks really good! I only reviewed related changes since I assumed the style updates are being reviewed in another PR.

The only thing I found is that the pet pictures became pretty small on a smaller screen. It would be nice to make it more responsive (like placing the picture above the news text on small screens).
Screen Shot 2022-03-02 at 8 49 22 PM

We can also add the pet's name and maybe the shelter name to the news updates, but I think it's optional.

File: src/routes/HomePage/HomePage.js (hunk 1 above)
Comment (dotte-osu) [reply src/routes/HomePage/HomePage.js:18]: changed!

**Comment by dotte-osu** [comment]
@asching7108  Added the shelter name and pet's name to the news updates. Fixed the mobile view.
image
image

**Review by asching7108 — APPROVED** [review]
Looks great!

**Pull request merged**